clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/InvocationHandler.java

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+package com.microsoft.aspnet.signalr;
+
+import java.util.ArrayList;
+
+public class InvocationHandler {
+    InvocationHandler(ActionBase action, ArrayList<Class<?>> classes) {
+        this.action = action;
+        this.classes = classes;
+    }
+
+    public ArrayList<Class<?>> classes;
+    public ActionBase action;
```

Comment:
Public fields? We should use the same pattern we use in C# (but Javaized) and have private fields with `getClasses`/`setClasses` (as needed), etc.

Reply:
We do it for completely private types in C#, so I don't see why we shouldn't in Java.